<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCadastroProfessor</class>
 <widget class="QDialog" name="FormCadastroProfessor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>580</y>
     <width>168</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_gravar">
      <property name="text">
       <string>Gravar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_cancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>381</width>
     <height>567</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>CPF</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cpf_cadastro"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nome_cadastro"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Logradouro</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="logradouro_cadastro"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Setor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="setor_cadastro"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Cidade</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cidade_cadastro"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Estado</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="estado_cadastro"/>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Celular</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="celular_cadastro"/>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>E-mail</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="email_cadastro"/>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Graduação</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="graduacao_cadastro"/>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Titulação</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="titulaco_cadastro"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
